docs/src/main/asciidoc/_galleon/Galleon_provisioning.adoc

```diff
@@ -0,0 +1,158 @@
+are deployed in public maven repositories. 
+
+When Galleon is used to install WildFly, WildFly feature-packs are retrieved and 
+their content is assembled to create an installation.
+
+== Installing WildFly using Galleon
+
+The Galleon maven plugin or Galleon CLI are used to install WildFly. The latest CLI and documentation can be downloaded 
+from the Galleon releases https://github.com/wildfly/galleon/releases[page].
+
+To install the latest final version of WildFly into the directory _my-wildfly-server_ call:
+
+[source,options="nowrap"]
+----
+galleon.sh install wildfly:current --dir=my-wildfly-server
+----
+
+Once installed, the directory _my-wildfly-server_ contains all that is expected to run a complete WildFly server. 
+By default, all standalone and domain configurations are installed.
+
+[NOTE]
+====
+
+Append the release version to install an identified release, for example:
+[source,options="nowrap"]
+----
+galleon.sh install wildfly:current#21.0.0.Final --dir=my-wildfly-server
+----
+====
+
+== Selecting the configurations to install
+
+Galleon allows you to specify only a subset of the default configurations to be installed.
+
+To only install the standalone.xml configuration call:
+
+[source,options="nowrap"]
+----
+galleon.sh install wildfly:current --dir=my-wildfly-server --default-configs=standalone/standalone.xml
+----
+
+To install multiple configurations, a comma separated list of default configurations can be provided, 
+for example:
+[source,options="nowrap"]
+----
+galleon.sh install wildfly:current --dir=my-wildfly-server --default-configs=standalone/standalone.xml,domain/domain.xml
+----
+
+[NOTE]
+====
+
+A default configuration is identified by _<configuration model>_/_<configuration name>_. WildFly defines _standalone_ and _domain_ models. 
+The _configuration name_ is the XML configuration file name (e.g.: _standalone.xml_, _domain.xml_, _standalone-ha.xml_).
+====
+
+[[wildfly_galleon_layers]]
+== WildFly Galleon layers
+
+WildFly Galleon feature-packs expose some link:++https://docs.wildfly.org/galleon/#_layers++[Galleon layers]. 
+A layer identifies one or more server capabilities that can be installed on its own or
+in combination with other layers. For example, if your application (e.g.: microservice) 
+is only making use of _JAX-RS_, Eclipse MicroProfile Config and _CDI_ server capabilities, you can choose to only install
+the _jaxrs_, _microprofile-config_ and _cdi_ layers. The _standalone.xml_ configuration would then only contain the 
+required subsystems and their dependencies.
+
+The benefit to installing WildFly using Galleon layers, in addition to configuration 
+trim down, is that Galleon only installs the needed content (JBoss Modules modules, scripts, etc.).
+
+To install a server only configured with _JAX-RS_ and _CDI_ capabilities call:
+
+[source,options="nowrap"]
+----
+galleon.sh install wildfly:current --dir=my-wildfly-server --layers=cdi,jaxrs
```

Comment:
Also examples such as this I think should primarily be based on the layers in the "Server Galleon layers" section - we recommend that is where to start but then show examples using the basic layers.

Reply:
We have both, I think it is fine to use basic layers, they offer more trimming.